dotnet/src/SemanticKernel.Core/Text/TextChunker.cs

```diff
@@ -32,12 +32,13 @@ public static class TextChunker
     /// <param name="text">Text to split</param>
     /// <param name="maxTokensPerLine">Maximum number of tokens per line.</param>
     /// <param name="tokenCounter">Function to count tokens in a string. If not supplied, the default counter will be used.</param>
+    /// <param name="splitOptions">Options to split string. If not supplied, the default options will be used.</param>
     /// <returns>List of lines.</returns>
-    public static List<string> SplitPlainTextLines(string text, int maxTokensPerLine, TokenCounter? tokenCounter = null)
+    public static List<string> SplitPlainTextLines(string text, int maxTokensPerLine, TokenCounter? tokenCounter = null, string[]? splitOptions = null)
```

Comment:
This would be a kind of breaking change, if we're happy to accept breaking changes, think maybe we could keep the `tokenCounter` as the last parameter since it seems would be used less frequently

```suggestion
    public static List<string> SplitPlainTextLines(string text, int maxTokensPerLine, string[]? splitOptions = null, TokenCounter? tokenCounter = null)
```